### PR TITLE
DOCS-8951-performance-impact-clarification-SE-17482

### PR DESCRIPTION
**Pull request by sarathecoubian**

### CODE REVIEW

**Review by michael-lee-mulesoft — DISMISSED** [review]

File: modules/ROOT/pages/configure-monitoring-cloudhub.adoc [hunk 1]
```diff
@@ -114,8 +114,9 @@ Use the Anypoint CLI and set the following property during your CI/CD deployment
 
 == Performance Impact
 
-Enabling monitoring can impact CPU utilization and memory utilization, which varies depending upon specific application configuration. You should test any performance impact in your pre-production environment prior to enabling monitoring in your production environment to ensure optimal performance and sizing.
-
+Enabling Anypoint Monitoring can impact CPU utilization and memory utilization, which varies depending upon specific application behavior, such as inbound and outbound call rates.
+For a typical application, expected CPU utilization impact is less than 5%.
+For an accurate impact analysis specific to your application, MuleSoft recommends that you test and estimate any performance impact in your pre-production environment prior to enabling monitoring in your production environment.
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/configure-monitoring-cloudhub.adoc:119]: Per the style guide we should avoid "MuleSoft recommends". Recommended fix can be: 
_...application, it's best to test and estimate..._
or either avoid the recommendation and make it:
_...application, test and estimate..._

**Review by Cristian-Venticinque — DISMISSED** [review]
Please check comment before merging.

File: modules/ROOT/pages/configure-monitoring-cloudhub.adoc (hunk 1 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/configure-monitoring-cloudhub.adoc:119]:
```suggestion
For an accurate impact analysis specific to your application, test and estimate any performance impact in your pre-production environment prior to enabling monitoring in your production environment.
```

**Review by hannanelson — APPROVED** [review]
LGTM

**Pull request merged**